api/block/schema.py

```diff
@@ -15,11 +15,18 @@
 
 
 class Block(SQLAlchemyObjectType):
+    """
+        Returns the payload with the fields
+        (id, name, officeId, state, offices, floors)
+    """
     class Meta:
         model = BlockModel
 
 
 class CreateBlock(graphene.Mutation):
+    '''
+        Returned payload after creating a block
```

Comment:
Hi @TeamoreA , Kindly look at the tense used in the docstrings and be consistent.